Xamarin.Forms.Core/Items/ItemsView.cs

```diff
@@ -114,8 +130,15 @@ public ItemSizingStrategy ItemSizingStrategy
 			OnScrollToRequested(new ScrollToRequestEventArgs(item, group, position, animate));
 		}
 
+		public void SendRemainingItemsThresholdReached()
```

Comment:
Does this one need `[EditorBrowsable(EditorBrowsableState.Never)]`?

Reply:
No.